pkg/constants/constants.go

```diff
@@ -187,16 +197,28 @@ var (
 	AutoScalerMetricsMemory AutoscalerMetricsType = "memory"
 )
 
+// Autoscaler Prometheus metrics
+var (
+	AutoScalerMetricsPrometheus AutoscalerMetricsType = "prometheus"
+)
+
+// Autoscaler Prometheus metrics
+var (
+	AutoScalerMetricsGraphite AutoscalerMetricsType = "graphite"
```

Comment:
You can put `AutoScalerMetricsGraphite` together with AutoScalerMetricsPrometheus within one var block.